pkg/ui/react-app/package.json

```diff
@@ -45,7 +45,8 @@
     "tempusdominus-bootstrap-4": "^5.1.2",
     "tempusdominus-core": "^5.0.3",
     "typescript": "^3.3.3",
-    "use-query-params": "^1.1.6"
+    "use-query-params": "^1.1.6",
+    "semantic-ui-react": "^2.0.1"
```

Comment:
Using a high level UI library just for tooltips is a bit too much. It will [increase the bundle size by a lot](https://bundlephobia.com/result?p=semantic-ui-react@2.0.1) too.

We are already using Reactstrap all over. Use https://reactstrap.github.io/components/tooltips/